pom.xml

```diff
@@ -206,6 +222,42 @@
               </execution>
             </executions>
           </plugin>
+
+        </plugins>
+      </build>
+    </profile>
+
+    <profile>
+      <id>integration-tests</id>
+      <build>
+        <plugins>
+          <!-- Integration Test Configuration -->
+          <plugin>
+            <groupId>org.apache.maven.plugins</groupId>
+            <artifactId>maven-failsafe-plugin</artifactId>
+            <version>3.3.0</version>
```

Comment:
don't hardcode the version here either ;-)

Reply:
Oh, wait surefire was already like that. Should I do it for every plugin then?

Reply:
just the ones that are used multiple times in the pom. The idea is to not have to hunt down all the instances of the plugin when updating it.